django/templatetags/static.py

```diff
@@ -150,5 +156,14 @@ def do_static(parser, token):
     return StaticNode.handle_token(parser, token)
 
 
+@register.simple_tag
 def static(path):
+    """
+    Return ether a StaticNode or staticfiles storage url if configured.
+
+    :param path: relative path to static asset
```

Comment:
We don't typically use this style of docstring. I prefer if you restructure it as a sentence.